web/src/components/vis/analyses.js

```diff
@@ -1,28 +1,24 @@
 import WilcoxonPlotTile from './WilcoxonPlotTile.vue';
 import AnovaTableTile from './AnovaTableTile.vue';
-import { wilcoxon_zero_methods, wilcoxon_alternatives, plot_types } from '../../utils/constants';
+import { plot_types } from '../../utils/constants';
 
 export default [
   {
     path: 'wilcoxon',
-    name: 'Wilcoxon signed-rank test',
+    name: 'Wilcoxon test',
     shortName: 'Wilcoxon Test',
-    description: 'The Wilcoxon signed-rank test tests the null hypothesis that two related paired samples come from the same distribution. In particular, it tests whether the distribution of the differences x - y is symmetric about zero. It is a non-parametric version of the paired T-test.',
+    description: 'TODO R custom code',
```

Comment:
Should this TODO be changed?

Reply:
well the only one who can write a proper text are our collaborators